euphoria-core/src/main/java/cz/seznam/euphoria/core/client/functional/BinaryFunction.java

```diff
@@ -19,12 +19,23 @@
 
 /**
  * Function of two arguments.
+ *
+ * @param <LEFT> the type of the left operand
+ * @param <RIGHT> the type of the right operand
+ * @param <OUT> the type of the result of the function application
  */
 @FunctionalInterface
 public interface BinaryFunction<LEFT, RIGHT, OUT> extends Serializable {
 
 
-  /** Return the result of this function. */
+  /**
+   * Return the result of this function.
```

Comment:
`This function returns result of this function`, cpt. obvious :smile: 
We can omit that.